### ISSUE TITLE
Investigate Backend options

### ISSUE THREAD

**Issue by codehag** [tc39/tc39.github.io#9]
Requirements: 

- [ ] Needs to make use of .md files for ease of editing
- [ ] shippable with github pages

Currently using jekyll. What are some other solutions, what are the benefits and problems with them? (e.g. https://www.11ty.io/)

**Comment by iddan** [comment]
[Gatsby](https://www.gatsbyjs.org/) is a great option: it provides excellent support for MD files, written in JS and embraces state of the art practices while performing as well as any site generator

**Comment by zoepage** [comment]
back to https://www.11ty.io/ maybe as this works with `.md` as well as `.html`?

**Comment by codehag** [comment]
list of generators also: https://www.staticgen.com/

**Comment by xtuc** [comment]
In case you're not aware, we could use https://docusaurus.io/. It has some limitations when wanting client-side interactions but very simple/good for a static website. 

It's markdown based.

**Comment by codehag** [comment]
Docusaurus was also one that i was looking at. it does use react for templating, so it has a similar drawback to gatsby -- perhaps i am also overthinking this optics issue. I am open to comments on this

For me using react and also graph ql at this point is over kill

**Comment by xtuc** [comment]
I might be wrong but Docusaurus generates HTML from the React app. I assumed that gatsby would too.

**Comment by codehag** [comment]
Yes, that is correct - I would like to see some more options before we settle on gatsby. I definitely had a shock suddenly seeing react and graphql in the application. so that is why I had the reaction that I did. I was expecting something that was just html and css, with maybe a little js to make templating.

I reached out to a few people who have a lot of experience running static sites, hopefully they will get back to me soon about their setup and what walls they hit. we don't have to fully rule out everything with react but I would strongly prefer a more neutral framework, or something that can be used in a more neutral way. For example if we use react to get the information for the table -- this is fine. But I would like to make that decision as a group and not be forced into it by a technology. Hopefully this makes sense

**Comment by ljharb** [comment]
Why is using react a drawback?

**Comment by zoepage** [comment]
@ljharb react is a great framework. Using it in our particular use case and context might make a statement towards a specific technology that might be missunderstood. We try to be as neutral as possible, this is why we decided in a meeting not to go with it.

**Comment by codehag** [comment]
@ljharb: to add to what @zoepage mentioned: this site should be an mvp - a static page with text content. There is no need for JavaScript at all, just content and styling. We might choose to add JavaScript at some point, but this is not in scope right now, and I would like to avoid being forced into a decision by the MVP.

Gatsby would effectively make us choose a framework (and also JSX, graphql, etc) when we do not need one, and there are a lot of static site generators which would work just fine with HTML templates.

**Comment by codehag** [comment]
I had a chat with ljharb, and i will just repeat what we discussed there:
He had three bullet points:
- the site should use js/css/html and no other langauges
- if a JS framework is used he would prefer react
- He does not see avoiding react to avoid "blessing" something as valid

However we did find a common ground. My points were
- I am fine if react is used in a measured, reasoned way (ie, not building the static site with it)
- since this is a content site i would prefer to keep it as simple as possible for the time being
- I do not want to get forced into a decision about a JS framework this early on, since there is no need for it

He mentioned some other points which I would like to echo, like JSX having quality linters, and html not having as well known linters. We might want to investigate this further. So far I have found htmllint and htmlhint.
